modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.tsx

```diff
@@ -7,11 +7,32 @@ import ClayButton from '@clayui/button';
 import ClayDropDown from '@clayui/drop-down';
 import ClayIcon from '@clayui/icon';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React, {useContext, useState} from 'react';
 
 import FrontendDataSetContext from '../../FrontendDataSetContext';
-import {triggerAction} from '../../utils/actionItems/index';
+import {triggerAction} from '../../utils/actionItems/triggerAction';
+
+export interface ICreationActionItem {
+	data?: {
+		permissionKey?: string;
+		size?: string;
+		title?: string;
+		type?: string;
+	};
+	href: string;
+	icon?: string;
+	label: string;
+	onClick: Function;
+	target:
+		| 'event'
+		| 'link'
+		| 'modal'
+		| 'modal-full-screen'
+		| 'modal-lg'
+		| 'modal-sm'
+		| 'sidePanel';
+	type: string;
```

Comment:
Added this `type` property because I've found that Objects is returning it for their creationMenu

Reply:
Yep, we are not using it, just trying to prevent future errors.